### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by AnelisaDee**
**Portfolio: add a contact form and finish the mobile version**
This project is developed using best practice of linters. I created the sections of this mobile website using the template found in Figma App.

**Project Requirements**
- Added a contact form to the portfolio using Flexbox.
- Added HTML validation to the form.
- Added 2 space indentation inside the .css file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -211,5 +211,23 @@ <h1>About me</h1>
         </div>
       </div>
     </section>
+      <div class="contact-me">
+        <h1 class="contact-heading">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>
```
Comment (leonardodiasb) [inline index.html]: - [ ] The heading of the contact section should follow the heading structure and hierarchy of the page and should not be an `h1`. There should be no more than one `h1` tag on the page and it should be the main title. Kindly change that to an `h2` tag and take a look [at this article](https://www.reliablesoft.net/h1-tag/) for more information. 👍

File: index.html [hunk 2]
```diff
@@ -211,5 +211,23 @@ <h1>About me</h1>
...
     </section>
+      <div class="contact-me">
+        <h1 class="contact-heading">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </h1>
+        <form action="https://formspree.io/f/meqdprbd" class="contact-form" method="post">
+          <input type="text" class="contact-form-text" maxlength="30" placeholder="Full name" required>
+          <input type="email" class="contact-form-email" placeholder="Email address" required>
+          <textarea name="contact" class="contact-textarea" maxlength="500" placeholder="Enter your text here" required></textarea>
+          <div class="form-button"><input type="button" value="Get In Touch" class="contact-button"></div>
```
Comment (leonardodiasb) [inline index.html]: - [ ] The form is not submitting properly due to the type of the input you added. Kindly change the type of the input button to `submit` so the form can be submitted.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @AnelisaDee, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of Github flow. 👍
 - Nice UI design. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi Team, 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**